services/identity/struct.go

```diff
@@ -0,0 +1,234 @@
+package identity
+
+import (
+	"encoding/binary"
+	"sort"
+
+	"fmt"
+	"strings"
+
+	"github.com/dedis/cothority/crypto"
+	"github.com/dedis/cothority/log"
+	"github.com/dedis/cothority/network"
+	"github.com/dedis/cothority/sda"
+	"github.com/dedis/cothority/services/skipchain"
+	"github.com/dedis/crypto/abstract"
+)
+
+// ID represents one skipblock and corresponds to its Hash
+type ID skipchain.SkipBlockID
+
+// Config holds the information about all devices and the data stored in this
+// identity-blockchain
+type Config struct {
+	Threshold int
+	Device    map[string]*Device
+	Data      map[string]string
+}
+
+// Device has write-access to the IdentityList if the threshold is given
+type Device struct {
+	Point abstract.Point
+}
+
+// NewConfig returns a new List with the first owner initialised
+func NewConfig(threshold int, pub abstract.Point, owner string) *Config {
+	return &Config{
+		Threshold: threshold,
+		Device:    map[string]*Device{owner: {pub}},
+		Data:      make(map[string]string),
+	}
+}
+
+// Copy makes a deep copy of the AccountList
```

Comment:
`Copy returns a deep copy of the AccountList` (yeah I know it's small, but it's everyone's right to nitpick !! :D)